js/src/setup-ads/ads-stepper/setup-accounts/index.js

```diff
@@ -42,20 +42,22 @@ const SetupAccounts = ( props ) => {
 				) }
 			/>
 			<Section
-				title={ __( 'Google account', 'google-listings-and-ads' ) }
+				title={ __( 'Connect accounts', 'google-listings-and-ads' ) }
 				description={ __(
-					'WooCommerce uses your Google account to sync with Google Merchant Center and Google Ads.',
+					'Any campaigns created through this app will appear in your Google Ads account. You will be billed directly through Google.',
 					'google-listings-and-ads'
 				) }
 			>
-				<AccountCard
-					className="gla-setup-ads-accounts__google-card"
-					appearance={ { title: google.email } }
-					description={ __(
-						'This Google account is connected to your store’s product feed.',
-						'google-listings-and-ads'
-					) }
-				/>
+				<VerticalGapLayout size="large">
```

Comment:
💡  Since the Google Ads card will be merged into this section in subsequent PR, the `<VerticalGapLayout>` was added here in advance.